File: 2000-reverse-prefix-of-word/2000-reverse-prefix-of-word.java
class Solution {
    public String reversePrefix(String word, char ch) {
        int idx=word.indexOf(ch);
        if(idx<=0)return word;
        StringBuilder s=new StringBuilder(word.substring(0,idx+1));
        StringBuilder str=s.reverse();
        str.append(word.substring(idx+1));
        return str.toString();
    }
}

File: 2000-reverse-prefix-of-word/2000-reverse-prefix-of-word.py
class Solution:
    def reversePrefix(self, word: str, ch: str) -> str:
        idx=word.find(ch)
        s=""
        if idx!=-1:
            s=word[idx:0:-1]
            s+=word[0]
        st=s+word[idx+1:]
        return st